tests/network/setup_multimachine.pm

```diff
@@ -30,13 +32,48 @@ sub run {
 
     # Configure static network, disable firewall
     disable_and_stop_service($self->firewall);
+    #disable apparmor
+    script_run("systemctl disable apparmor.service");
+    script_run("systemctl stop apparmor.service");
 
     # Configure the internal network an  try it
     if ($hostname =~ /server|master/) {
         setup_static_mm_network('10.0.2.101/24');
+        #if server running opensuse.
+        #my $base_product_opensuse = get_var('DISTRI');
+        #if ($base_product_opensuse eq "opensuse") {
+        if (is_opensuse) {
+            assert_script_run 'systemctl stop NetworkManager';
+            assert_script_run 'systemctl disable NetworkManager';
+            assert_script_run 'systemctl start  wicked';
+            assert_script_run('echo "195.135.221.150 openqa.opensuse.org" >> /etc/hosts');
```

Comment:
Why do you do this?

Reply:
some times on TW, NetworkManager not accept MM ip address,  I stop NetworkManager and start wicked. And in my openqa VM, TW not solve openqa.opensuse.org DNS name, fix on /etc/hosts solve quickly, not elegant, but solved.

Reply:
Mmm I understand but this is strange :( Why do you btw. need `openqa.opensuse.org` and can you also look at what you have on `/etc/resolv.conf` file?

Reply:
Ok, I beliave solve with /etc/resolv.conf. Thanks Pavel.